### ISSUE TITLE
Missing command in instructions for adding tab completions to zsh

### ISSUE THREAD

**Issue by jsdodge** [python-poetry/poetry#7966]
- [x ] I have searched the [issues](https://github.com/python-poetry/poetry/issues) of this repo and believe that this is not a duplicate.

## Issue
Following the instructions for enabling tab completions on [`zsh`](https://python-poetry.org/docs/#zsh) produce a `zsh: no such file or directory` error. The instructions returned by `poetry help completions` include additional commands, which execute without error:
```zsh
  ZSH:
  
  ZSH completions are commonly stored in any directory listed in your `$fpath` variable. To use these completions, you must either add the generated script to one of those directories, or add your own to this list.
  
  Adding a custom directory is often the safest best if you're unsure of which directory to use. First create the directory, for this example we'll create a hidden directory inside our `$HOME` directory
  
  `mkdir ~/.zfunc`
  
  Then add the following lines to your `.zshrc` just before `compinit`
  
  `fpath+=~/.zfunc`
  
  Now you can install the completions script using the following command
  
  `poetry completions zsh > ~/.zfunc/_poetry`
  
  You must then either log out and log back in, or simply run
  
  `exec zsh`
  
  For the new completions to take affect.
  
```

**Comment by Secrus** [comment]
This is not a Poetry issue. Your shell is misconfigured. Our instruction is generic and expects you to handle shell config yourself.

**Comment by jsdodge** [comment]
Forgive my persistence, but I do not believe that the error occurred because my shell was misconfigured. The error occurred when I followed the [online instructions](https://python-poetry.org/docs/#zsh), which direct the user to use
```
poetry completions zsh > ~/.zfunc/_poetry
```
then add the following lines to `~/.zshrc`, if they do not already exist:
```
fpath+=~/.zfunc
autoload -Uz compinit && compinit
```

When I followed these instructions, I received an error that `~/.zfunc/_poetry` did not exist. The reason I think this is not due to misconfiguration of my shell is that the instructions provided by `poetry help completions`  include the following lines, which were all I needed to avoid the error:

>   Adding a custom directory is often the safest best if you're unsure of which directory to use. First create the directory, for this example we'll create a hidden directory inside our `$HOME` directory
>   
>   `mkdir ~/.zfunc`

I am suggesting that these command-line instructions be included in the online instructions as well.
